doc/dev_all.dox

```diff
@@ -145,18 +145,58 @@ Concrete device drivers that implement sets of interfaces.
 /**
 \ingroup dev_impl
 
-\defgroup dev_impl_wrapper Network Servers (Wrappers)
+\defgroup dev_impl_wrapper Network Servers (Wrappers - pre NWC/NWS architecture)
```

Comment:
Is `dev_impl_wrapper` group still used somewhere? Or we can remove it?

Reply:
It is still used in old wrapper devices that will be replaced by their `nws` version and deprecated

Reply:
Ok, thanks!